flycheck.el

```diff
@@ -9067,6 +9067,18 @@ Requires Sphinx 1.2 or newer.  See URL `http://sphinx-doc.org'."
   :predicate (lambda () (and (flycheck-buffer-saved-p)
                              (flycheck-locate-sphinx-source-directory))))
 
+(defun flycheck-ruby--find-project-root (_checker)
+  "Come up with a suitable default directory to run CHECKER in.
+
+This will either be the directory that contains `Gemfile' or,
+if no such file is found in the directory hierarchy, Flycheck will
+use current value of `default-directory'."
```

Comment:
Please add 'the' between 'use' and 'value' here.

Reply:
Changed. 

If you really need tests I'll ad them, but it feels more like writing tests for rubocop than for flycheck in this case.